.github/workflows/docathon-label-sync.yml

```diff
@@ -0,0 +1,33 @@
+name: Docathon Labels Sync
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited]
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if PR mentions an issue and get labels
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install dependencies
+        run: |
+          pip install requests
+          pip install PyGithub
+      - name: Run Python script
```

Comment:
It's the same the previous step, remove?

Reply:
resolved